example/material_card_example/build.yaml

```diff
@@ -0,0 +1,5 @@
+targets:
+  $default:
+    builders:
+      angular_components|scss_builder:
+        enabled: True
```

Comment:
True -> true?

Reply:
I have seen uppercased booleans in other build.yaml files. Not sure if there is history or a reason or where it originally started. Is there guidance somewhere?

Reply:
Just read up on that: apparently case doesn't matter. I'm not aware of any guidelines anywhere.